### PR TITLE
chore: refine the connection description for SDKs

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

as title

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by fengjiachun — APPROVED** [review]
LGTM

File: docs/user-guide/ingest-data/for-iot/grpc-sdks/go.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ cfg := greptime.NewConfig("127.0.0.1").
     // Enable secure connection if your server is secured by TLS
     // WithInsecure(false).
     // set authentication information
+    // If the database doesn't require authentication, just leave the username and password empty
```
Comment (sunng87) [inline docs/user-guide/ingest-data/for-iot/grpc-sdks/go.md:46]: Don't leave it empty, just remote this `WithAuth`

**Review by sunng87 — COMMENTED** [review]
Do we need to update Java example as well?

**Comment by nicecui** [comment]
@sunng87 PTAL

**Review by sunng87 — APPROVED** [review]

**Pull request merged**